### ISSUE TITLE
DNS resolution via dig doesn't work

### ISSUE THREAD

**Issue by cathugger** [loki-project/loki-network#37]
```
# dig i4irznec3pkdh7gay6xsmkyyqag4q8643kut739by17cuiwdnxqo.loki @127.0.0.1
;; Warning: Message parser reports malformed message packet.

; <<>> DiG 9.11.2-P1 <<>> i4irznec3pkdh7gay6xsmkyyqag4q8643kut739by17cuiwdnxqo.loki @127.0.0.1
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 13048
;; flags: qr; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 0
;; WARNING: Message has 1 extra bytes at end

;; QUESTION SECTION:
;i4irznec3pkdh7gay6xsmkyyqag4q8643kut739by17cuiwdnxqo.loki. IN A

;; Query time: 1922 msec
;; SERVER: 127.0.0.1#53(127.0.0.1)
;; WHEN: Thu Nov 01 19:55:44 UTC 2018
;; MSG SIZE  rcvd: 145
```

**Comment by majestrate** [comment]
`;; WARNING: Message has 1 extra bytes at end`
that's a good hint for @neuroscr

**Comment by cathugger** [comment]
Doesn't work with `drill` either. Using current git master.

**Comment by majestrate** [comment]
it seems fixed at the moment

**Comment by majestrate** [comment]
fixed by #103

**Comment by despair86** [comment]
can confirm, `nslookup` on windows returns proper results without resubmission
